app/src/main/res/values/styles.xml

```diff
@@ -33,6 +34,11 @@
     <style name="Theme.Sunflower.NoActionBar">
         <item name="windowActionBar">false</item>
         <item name="windowNoTitle">true</item>
+        <item name="android:navigationBarColor">@android:color/transparent</item>
+        <item name="android:statusBarColor">@android:color/transparent</item>
+        <item name="windowActionBarOverlay">true</item>
+        <item name="android:windowTranslucentStatus">true</item>
```

Comment:
De we want these translucent flags? I thought they were a KitKat thing not needed any longer.